docs/architecture/data_load.rst

```diff
@@ -1,50 +0,0 @@
-API
-===
-Datacube Load
--------------
-The basics of the datacube load function are:
-
-* `Find Datasets`_ - Look up the index​
-* `Group Datasets`_ - Sort and group by time​ (and/or other dimensions)
-* `Load Data`_ - Read from storage, write to output array
-
-Find Datasets
-~~~~~~~~~~~~~
-Base on the user query, look up the index to find a list of `dataset` objects.
-
-Group Datasets
-~~~~~~~~~~~~~~
-From the query we end up with a list of datacube Dataset objects.
-
-We sort these over the non-spatial dimensions.​
-
-(Currently the time dimension is hard-coded in a few places,
-but the design allows for extension to arbitrary dimensions.)​
-
-This can be the time stamp of the dataset, or a `solar_day` aggregation that group together based on time and
-longitude, to avoid splitting days at UTC midnight.​
-
-Load Data
-~~~~~~~~~
-For each group we created in the last step:​
-    For each measurement (band):​
-        For each dataset in the final chunk​
```

Comment:
I'm pretty sure it goes over bands first, hence the problems raised in this issue #383